### ISSUE TITLE
Add svelte/attributes-line-break

### ISSUE THREAD

**Issue by jamesst20** [sveltejs/eslint-plugin-svelte#762]
### Motivation

There are currently 2 existing rules that helps formatting attributes line break:

* "svelte/first-attribute-linebreak"
* "svelte/max-attributes-per-line"

The problem with them is that there is no way to consider the *ligne length* to enforce proper style.

### Description

`"svelte/attributes-line-break": ["error", { printWidth: 90 }]`

```
// Good

// 41 characters
<MyComponent attr1="test" attr2="test" />

// 99 characters
<MyComponent
  attr1="test"
  attr2="test"
  attr3="some very long text that takes spaces and many..."
/>


// Invalid

// 41 characters
<MyComponent
  attr1="test"
  attr2="test" />

// 99 characters
<MyComponent  attr1="test"  attr2="test"  attr3="some very long text that takes spaces and many..."
/>
```

### Examples

```svelte
<script>
</script>

<!-- ✓ GOOD -->
<Foo />
<Foo />
<Foo />

<!-- ✗ BAD -->
<Foo />
<Foo />
<Foo />
```

### Additional comments

These are only examples, I am not very much opiniated on the rule name or customization as long as it can give the proper result in the end.

Right now Prettier is able to enforce this style correctly, however Prettier is very opiniated and doesn't offer much customization and i would like to get rid of it.

**Comment by ota-meshi** [comment]
Thank you for the rule suggestion.
I'm interested in that rule, but I think it conflicts with `svelte/max-attributes-per-line`.
Could you please consider changing the rule so that it doesn't conflict, or consider deprecating `svelte/max-attributes-per-line` and creating a new rule that includes the use case of `svelte/max-attributes-per-line`?

https://sveltejs.github.io/eslint-plugin-svelte/rules/max-attributes-per-line/

**Comment by jamesst20** [comment]
> Thank you for the rule suggestion. I'm interested in that rule, but I think it conflicts with `svelte/max-attributes-per-line`. Could you please consider changing the rule so that it doesn't conflict, or consider deprecating `svelte/max-attributes-per-line` and creating a new rule that includes the use case of `svelte/max-attributes-per-line`?
> 
> https://sveltejs.github.io/eslint-plugin-svelte/rules/max-attributes-per-line/

Indeed I'm open to all this. I currently have these 2 rules disabled in favor of Prettier for the time being until anyone implement this suggestion :)

Do you have a proposal? I could edit the top post

**Comment by baseballyama** [comment]
I don’t think eslint-plugin-svelte needs to support rules related to styling.
It might be better to leave everything to Prettier.

**Comment by ota-meshi** [comment]
In my opinion, support for stylistic rules is still needed.
Prettier is opinionated, so it might not be good for some projects.
ESLint is not a formatter, but the ESLint rule options are flexible, so I think the stylistic rules implemented in the ESLint plugin are still a good option for some projects.
